src/Pim/Component/Catalog/Normalizer/Indexing/Product/NumberNormalizer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing\Product;
+
+use Pim\Component\Catalog\AttributeTypes;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalizer for a decimal or an integer product value
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
```

Comment:
I don't know when we changed this rule be before there were only one author. I don't see the point of adding multiple